templateChecks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Also, I think this script should do `set -e` so any command yielding an error aborts the execution and marks it as unsuccessful. Otherwise, it will continue and the error state of the script will be determined by the last command. @matir91, bashman, thoughts?